quinn/src/connection.rs

```diff
@@ -638,6 +639,7 @@ where
     incoming_bi_streams_reader: Option<Waker>,
     datagram_reader: Option<Waker>,
     pub(crate) finishing: HashMap<StreamId, oneshot::Sender<Option<WriteError>>>,
+    pub(crate) stopped: HashMap<StreamId, oneshot::Sender<VarInt>>,
```

Comment:
To ensure blocked tasks wake up when the connection is lost (e.g. due to timeout), we need to drain this in `Connection::terminate`, similar to `blocked_readers`/`writers`.